src/AutoPilotPlugins/AutoPilotPlugin.cc

```diff
@@ -61,7 +61,7 @@ void AutoPilotPlugin::parametersReadyPreChecks(void)
     _recalcSetupComplete();
 
     // Connect signals in order to keep setupComplete up to date
-    for(const QVariant componentVariant: vehicleComponents()) {
+    for(QVariant componentVariant: vehicleComponents()) {
```

Comment:
Seems strange though that QVariantList doesn't support reference. I guess it is what it is.